web/forms.py

```
@@ -96,6 +98,25 @@
 ]
 
 
+class AccountDeleteForm(forms.Form):
+    password = forms.CharField(
+        label=_("Password"),
+        strip=False,
+        widget=forms.PasswordInput(attrs={"autocomplete": "current-password"}),
+        help_text=_("Enter your password to confirm account deletion."),
+    )
+
+    def __init__(self, user, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)